lib/ngrok/tunnel.rb

```diff
@@ -75,6 +75,7 @@ def ngrok_exec_params
         exec_params = "-log=stdout -log-level=debug "
         exec_params << "-authtoken=#{@params[:authtoken]} " if @params[:authtoken]
         exec_params << "-subdomain=#{@params[:subdomain]} " if @params[:subdomain]
+        exec_params << "-hostname=#{@params[:hostname]}" if @params[:hostname]
```

Comment:
Ah - yes, thanks. Missed that as my manual test wasn't setting the config option and I didn't notice it. Fixed.